server/utils/isUuid.ts

```
@@ -0,0 +1,5 @@
+const UUID_REGEX = /^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$/i;
```

ESLint

[error] Insert `⏎·`

(prettier/prettier)